aporia/README.md

```diff
@@ -0,0 +1,36 @@
+# Agent Check: Aporia
+
+## Overview
+
+Aporia is the ML observability platform, empowering ML teams with vigilant model monitoring and real-time alerts for drift, bias, performance, and data integrity while providing seamless metric tracking via tailor-made dashboards. Aporia's Production Investigation Room serves as a collaborative notebook, streamlining root cause analysis and explainability to unlock actionable insights for optimizing model performance.
+
+This integration allows Datadog users to report their Aporia alerts to Datadog for a unified monitoring system. Datadog users will be able to receive, view, and track live ML alerts from Aporia within their Datadog account. Users will be able to know exactly which model and monitor fired the alert, with specific configuration details and additional information about the alert. Within their Datadog account, users will be able to view and track alerts over time by their severity.
+
+## Setup
+
+### Installation
+
+First you'll need an Aporia account. Visit [Aporia][1] for more info and to book a demo.
+Once an account is available, [login to your Aporia account][2], then visit the Aporia integrations page and add a Datadog integration. This guides you through the Datadog OAuth2 flow to grant Aporia the necessary permissions to your Datadog account.
+
+### Configuration
+
+Once integrated, you can configure monitors in Aporia to send their Alerts to Datadog as well. You can track these alerts in Datadog and define follow-up actions as you see fit.

```

Comment:
```suggestion
Once integrated, you can configure monitors in Aporia to send alerts to Datadog as well. You can track these alerts in Datadog and define follow-up actions as you see fit.

```

Reply:
Can you specify if these alerts will be sent in as events? How will the user access them in Datadog?